dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -893,6 +908,8 @@ public void checkEntryFile(String localFilePath, List<String> argsList, String d
 
         if (ext.equals(Type.CWL)) {
             if (content.equals(Type.CWL)) {
+                validateCWL(localFilePath);
```

Comment:
This requires more documentation. `validateCWL` doesn't have a return type and doesn't throw exceptions. 
It isn't clear to me how it reports error messages or stops users from using invalid CWL files.